lib/ansible/executor/module_common.py

```diff
@@ -1288,7 +1289,10 @@ def modify_module(module_name, module_path, module_args, templar, task_vars=None
     return (b_module_data, module_style, shebang)
 
 
-def get_action_args_with_defaults(action, args, defaults, templar):
+def get_action_args_with_defaults(action, args, defaults, templar, collection_list=None):
```

Comment:
may be better to just pass task at this point

Reply:
yep, but tempted to just wait and revamp the whole system, this shoudl just give us a transition to collections